src/components/Card/Card.js

```diff
@@ -89,6 +89,14 @@ export default class Card extends Component {
           onClick={(e) => this.flipCard(e)}
         >
           <div className='card-front'>
+            <div className={`image-side-title card-title card-header-${cardType}`}>
+              <h1>{cardObj.name}</h1>
+              <i
+                className={`fas fa-star ${cardObj.favorite && 'favorite'} favorite-btn`}
+                onClick={() => this.toggleFavorite(cardObj)}
+              >
```

Comment:
Cool. Maybe we should consider adding a favorite component since this same code is used in several places now.  If we have time, we should implement that